AdminWebsite/AdminWebsite/ClientApp/src/app/booking/assign-judge/assign-judge.component.ts

```diff
@@ -86,7 +88,12 @@ export class AssignJudgeComponent extends BookingBaseComponent implements OnInit
         this.loadJudges();
         this.initForm();
 
-        this.invalidPattern = this.route.snapshot.data['emailPattern'];
+        this.$subscriptions.push(
```

Comment:
You don't need to use the subscriptions observable - just do:

```ts
this.configService
                .getClientSettings()
                .pipe(map(x => x.test_username_stem))
                .subscribe(x => (this.invalidPattern = x));
```

Reply:
Done